lib/libutils/ext/include/asm.S

```diff
@@ -90,3 +90,7 @@ UNWIND(	.fnend)
 	.macro END_DATA name
 	.size \name , .-\name
 	.endm
+
+#if defined(__aarch64__)
```

Comment:
Already taken mcare of in the `BTI` macro if I recall correctly, so remove.

Reply:
ok